Bluetooth: Mesh: add changelog for added metadata to mesh models (#16643)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Bluetooth Mesh
--------------

* Updated:

>  * Metadata has been added to the Light Lightness Server, Light HSL Server, Light CTL Temperature Server, the Sensor Server, and Time Server models in form of an optional parameter.
>    To use :kconfig:option:`CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` needs to be enabled.

Reviewer: ```suggestion
 * Added metadata as optional parameter for models Light Lightness Server, Light HSL Server, Light CTL Temperature Server, Sensor Server, and Time Server.
   To use the metadata, enable the :kconfig:option:`CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` Kconfig option.
```
